lib/player.rb

```diff
@@ -0,0 +1,28 @@
+class Player
+  attr_writer :moves
+  attr_reader :sign, :name, :moves
```

Comment:
Only if actually needed, the use of use attr_reader and attr_writer then attr_accesor is most.

# Great Job So Far

Kindly check your code and submit again for another review

### You're almost there

### Happy Coding :computer: :nerd_face: 

#### Daniel Laloush